Translate job offer to Spanish

sauloperez opened the pull request:
## Description

I translated the job offer to Spanish but I also took the chance to clean things up. When we first added the jobs page, we created a new layout to have separate styling. This made it possible to publish it pretty quickly but now it's time to pay the tech debt. Things are getting cumbersome.

We got one new page `/es/python_backend.html`.

## Screenshots

![screenshot from 2019-01-02 16-24-31](https://user-images.githubusercontent.com/762088/50598472-27b6d200-0eab-11e9-9ab4-9393c9459968.png)
![screenshot from 2019-01-02 16-24-17](https://user-images.githubusercontent.com/762088/50598473-27b6d200-0eab-11e9-8528-b6179b303a82.png)

sauloperez committed:
Fix navbar markup to remove nested <li>

sauloperez committed:
Extract hero_logo

This also removes the footer include since it was useless and stops
splitting </div> in separate files. Opening and closing tags should be
in the same file to avoid broken pages.

sauloperez committed:
Refactor and remove simple layout

naoisegolden commented on _includes/navbar_brand.html:
```diff
@@ -0,0 +1,4 @@
+<a class='navbar-brand' href="/">
+   <img width='300px' src='{{ "/assets/img/logo.svg" | prepend:site.baseurl }}')} />
```
```suggestion
   <img width='300' src='{{ "/assets/img/logo.svg" | prepend:site.baseurl }}')} />
```

`img` `width` and `height` attributes are ints, the size in pixels  but without the `px`.

naoisegolden reviewed (approved):
LGTM, only 1 correction

naoisegolden commented on _layouts/default.html:
```diff
@@ -3,9 +3,36 @@
+      {% include language_switcher.html %}
+
+      <ul class="navbar-nav ml-auto">
+        <li class="nav-item"><a href='/blog.html' class='nav-link'>Blog</a></li>
```
@sauloperez as far as I can see, there is no link to the jobs page. Is this by design or is it a mistake?

```suggestion
        <li class="nav-item"><a href='/blog.html' class='nav-link'>{% t pages.navigation.blog %}</a></li>
        <li class="nav-item"><a href='/jobs.html' class='nav-link'>{% t pages.navigation.jobs %}</a></li>
```

(If you accept this suggestion you'll still need to update the i18n files)

sauloperez replied:
Yes, still not there. That comes with the next commit. So far, only refactoring.

sauloperez committed:
Add translations for jobs index page

sauloperez committed:
Link job detail page

sauloperez committed:
Fix phrase in Catalan

sauloperez committed:
Translate job offer to Spanish

sauloperez committed:
Link jobs from menu and translate it

sauloperez committed:
Make all content 700px wide maximum

Not only the blog but all sections suffer from bad readability. This
will fix it.

sauloperez committed:
Have a single jobs index page with keys translated

sauloperez committed:
Use /jobs/python_backend as URL

naoisegolden committed:
Update _includes/navbar_brand.html

img width and height attributes are ints, the size in pixels but without the px.

Co-Authored-By: sauloperez <saulopefa@gmail.com>

alvaro-chalar commented on _i18n/es/python_backend.md:
```diff
@@ -0,0 +1,44 @@
+## Acerca de Coopdevs
+
+Coopdevs es una entidad que aporta software libre a la economía social i solidaria. Surgida desde profesionales del sector tecnológico de Barcelona con inquietudes sociales, proporciona amplios conocimientos técnicos i metodologías de trabajo en este ámbito económico. Entre muchos otros proyectos, impulsamos [Katuma](http://katuma.org/), formamos parte del equipo de desarrollo de [Open Food Netwo ...
```
`social y solidaria`, `conocimientos técnicos y metodologías`

alvaro-chalar commented on _i18n/es/python_backend.md:
```diff
@@ -0,0 +1,44 @@
+
+Ahora buscamos gente con espíritu cooperativo que tenga ganas de compartir este reto aportando su experiencia de desarrollo de software en un momento importante para hacer crecer el proyecto.
+
+La candidata se unirá principalmente al equipo que trabaja en la linea de negocio de ERP para cooperativas que se desarrolla en Python. Esta unidad se encarga de acompañar y ayudar a cooperativas y otras entidades a la sistematización de sus procesos internos y su optimización con software ERP. Las tareas són:
```
`línea`.....`las tareas son:`

alvaro-chalar commented on _i18n/es/python_backend.md:
```diff
@@ -0,0 +1,44 @@
+
+* Haber contribuido en proyectos open source
+* Interés en mejorar el ecosistema de la economía social y solidaria y la participación en proyectos de este ámbito
+* Experiencia con algun ERP escrito en Python
```
`algún`

alvaro-chalar commented on _i18n/es/python_backend.md:
```diff
@@ -0,0 +1,44 @@
+* Familiaridad con la línea de comandos
+* Interés en Data-Drive: basar las decisiones en datos empíricos a partir de sistemas de business intelligence, analíticas y monitorización
+
+<br/>
```
do you need this?

sauloperez replied:
I use it to separate this last sentence from the main text

alvaro-chalar commented on _i18n/es/python_backend.md:
```diff
@@ -0,0 +1,44 @@
+* Interés en Data-Drive: basar las decisiones en datos empíricos a partir de sistemas de business intelligence, analíticas y monitorización
+
+<br/>
+Aplica para esta oferta enviandonos un mail a <a href="mailto: info@coopdevs.org">info@coopdevs.org</a>
```
`enviándonos`

alvaro-chalar commented on _i18n/es/python_backend.md:
```diff
@@ -0,0 +1,44 @@
+<br/>
+Aplica para esta oferta enviandonos un mail a <a href="mailto: info@coopdevs.org">info@coopdevs.org</a>
+
+### Únete y impulsemos la economía social y solidaria juntos!
```
`Únete e impulsemos`

sauloperez committed:
Fix Spanish translation

alvaro-chalar reviewed (approved):
:clap:
